### DISCUSSION TITLE
Reports > getReportDocument data discrepancies versus Vendor Central

### DISCUSSION THREAD

**Discussion by vikadis** [amzn/selling-partner-api-models#3229] [General]
Hi, 
I am noticing that this endpoint is only giving me sales data for:
-  salesMetricsByProgram: Retail
And no data for:
- salesMetricsByProgram: Business
- salesMetricsByProgram: Fresh
- salesMetricsByProgram: PrimeNow

Even though when I download the reports through VendorCentral there are sales for platforms other than Retail.
I am using the same date range and other params

Thank you!

**Comment by Guerri114** [comment]
@vikadis : I suggest creating a Contact Us case in Vendor Central (if you are the vendor) or via the Contact Us link [here](https://developer.amazonservices.com/support) (if you are a 3rd party developer who works with vendors) to explore this issue in more detail. You'll want to include the full details of the request you are making, including the selling partner ID for the vendor account (it will look something like this: amzn1.vg.12345).
